app/src/main/res/values-in/strings.xml

```diff
@@ -0,0 +1,39 @@
+<resources>
+    <string name="app_name">MVVM Boilerplate</string>
+    <string name="action_ok">OK</string>
+    <string name="action_cancel">BATALKAN</string>
+    <string name="message_user_unautorizhed">Anda telah login pada device lain</string>
```

Comment:
this should not be used anymore, can you also move the english one as well?

Reply:
We can just remove this mas @fajarnuha @suryamudti this is from my boilerplate project 😅

Reply:
Okay, this will removed 👍